src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -169,7 +169,7 @@ exports[`Anchor icon label renders 1`] = `
       <svg
         color="#7D4CDB"
       />
-      <div
+      <span
```

Comment:
The fact that the div is not a child of span anymore is obviously the right approach, but I was wondering if you have more insights into why this change happened. 
I can see why Box will change its snapshots due to the new test, but I'm not sure why did this snapshot of Anchor change. I'll appreciate the insight.

Reply:
That's what I was alluding to in the second paragraph of the description. If you look at the original issue it was found because Anchor does this internally and that was what the submitter used as the example problem. This change fixes that and it shows up as the change in Anchor's output when you have an icon and label.